### DISCUSSION TITLE
Courseplay course sync?

### DISCUSSION THREAD

**Discussion by ToothyDev** [Courseplay/Courseplay_FS22#2469] [Q&A] [answered]
So, I made a course and saved it in the course manager. I found it's location in the game files and noticed that it was in a "Singleplayer" Subfolder in the map's folder, and doesn't appear on anyone elses course manager. Is it possible to somehow sync saved courses to others? I tried making a corresponding "Multiplayer" folder but it didn't show up ingame with any courses, so I wonder why its specifically called "Singleplayer" when I'm on a server

**Comment by Tensuko** [comment, marked as answer]
There is no course sync by us.
You can load it into a vehicle and let other user save it.
You can share your courses from folder.
I know there are tools to sync a folder with an online folder one, maybe if all do that it is a solution.
Sync course for MP is not on our TODO for now. Maybe it will happen at a later time if the bug stuff is done.
